File: wf-api-integration/references/beneficiary/remove/java/model/response/RemoveBeneficiaryResponse.java
package {basePackage}.wf.model.response;


import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;

/**
 * WorldFirst removeBeneficiary 响应对象。
 *
 */
public class RemoveBeneficiaryResponse {

    /** API 调用结果 */
    private Result result;

    /** 被删除的收款人令牌 */
    private String beneficiaryToken;

    public Result getResult() {
        return result;
    }

    public void setResult(Result result) {
        this.result = result;
    }

    public String getBeneficiaryToken() {
        return beneficiaryToken;
    }

    public void setBeneficiaryToken(String beneficiaryToken) {
        this.beneficiaryToken = beneficiaryToken;
    }

    public boolean isSuccess() {
        return result != null && "S".equals(result.getResultStatus());
    }

    @Override
    public String toString() {
        return ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE);
    }
}

File: wf-api-integration/references/beneficiary/remove/golang/model/response/remove_beneficiary_response.go
package response

// Result represents the standard WF API result object.
type Result struct {
	ResultStatus  string `json:"resultStatus"`
	ResultCode    string `json:"resultCode"`
	ResultMessage string `json:"resultMessage"`
}

// RemoveBeneficiaryResponse is the response for removeBeneficiary API.
type RemoveBeneficiaryResponse struct {
	Result           Result `json:"result"`
	BeneficiaryToken string `json:"beneficiaryToken,omitempty"`
}

